barretenberg/cpp/bootstrap.sh

```diff
@@ -54,23 +57,34 @@ echo "# Building with preset: $PRESET"
 echo "# When running cmake directly, remember to use: --build --preset $PRESET"
 echo "#################################"
 
-# Build native.
-cmake --preset $PRESET -DCMAKE_BUILD_TYPE=RelWithAssert
-cmake --build --preset $PRESET --target bb
+function build_native {
+  cmake --preset $PRESET -DCMAKE_BUILD_TYPE=RelWithAssert
+  cmake --build --preset $PRESET --target bb
+}
+
+function build_wasm {
+  cmake --preset wasm
+  cmake --build --preset wasm
+}
+
+function build_wasm_threads {
+  cmake --preset wasm-threads
+  cmake --build --preset wasm-threads
+}
+
+g="\033[32m"  # Green
+b="\033[34m"  # Blue
+p="\033[35m"  # Purple
+r="\033[0m"   # Reset
+
+(build_native > >(awk -v g="$g" -v r="$r" '$0=g"native: "r $0')) &
+(build_wasm > >(awk -v b="$b" -v r="$r" '$0=b"wasm: "r $0')) &
+(build_wasm_threads > >(awk -v p="$p" -v r="$r" '$0=p"wasm_threads: "r $0')) &
```

Comment:
Nice! Just one comment/concern: could we potentially have race conditions? Looking at the output, seems that the path of the created objects is either
* ...file.cpp.o (for native)
* ...file.cpp.obj (for wasm/wasm-threads)

If that's true, especially the last 2 could overwrite each other. (I haven't looked into full detail.)